### PR TITLE
Adjust `user_password_recovery.email` template for newer core implementation

### PR DESCRIPTION
**Pull request by sveneberth**
Related to https://github.com/viur-framework/viur-core/pull/682 
Resolves https://github.com/viur-framework/viur-base/issues/94

### CODE REVIEW

**Comment by ArneGudermann** [comment]
Should we output a minimal of the Useragent data?

https://github.com/viur-framework/viur-core/blob/338774eed482c15b12a9bc4abb1e0492ee806acf/src/viur/core/modules/user.py#L368-L370

File: deploy/emails/user_password_recovery.email [hunk 1]
```diff
@@ -6,16 +6,16 @@ Reset Password
...
+        open this link to confirm the password reset process:
+        <a href="{{ getHostUrl() }}/user/auth_userpassword/pwrecover?recovery_key={{ recovery_key }}">
+            {{ getHostUrl() }}/user/auth_userpassword/pwrecover?recovery_key={{ recovery_key }}
+        </a>
 
-		or enter the following verification code manually:
-		<pre>{{ skel["recoveryKey"] }}</pre>
-	</div>
+        or enter the following verification code manually:
+        <pre>{{ recovery_key }}</pre>
```
Comment (ArneGudermann) [inline deploy/emails/user_password_recovery.email:19]:
```suggestion
```
I think this is obsolete now?

**Comment by sveneberth** [comment]
> Should we output a minimal of the Useragent data?
> 
> https://github.com/viur-framework/viur-core/blob/338774eed482c15b12a9bc4abb1e0492ee806acf/src/viur/core/modules/user.py#L368-L370

That's the reason we have this... I have added.
@ArneGudermann Do we also get the IP and the (resulting) location in the core's user-agent dict?

And I have made the address even more personal, so that it does not look so 08/15 phishing.

**Comment by ArneGudermann** [comment]
> That's the reason we have this... I have added. @ArneGudermann Do we also get the IP and the (resulting) location in the core's user-agent dict?

We can get the IP from `current.request.get().request.remote_addr` but for the location I think we need a third party service.

File: deploy/emails/user_password_recovery.email [hunk 2]
```diff
@@ -2,20 +2,34 @@ Reset Password
...
+            <strong>Browser:</strong> <code>{{ user_agent["browser"] }}</code>
+        </p>
+
+        open this link to reset your password:
+        <a href="{{ getHostUrl() }}/user/auth_userpassword/pwrecover?recovery_key={{ recovery_key }}">
+            {{ getHostUrl() }}/user/auth_userpassword/pwrecover?recovery_key={{ recovery_key }}
+        </a>
+        or enter the following verification code manually:
+        <pre>{{ recovery_key }}</pre>
+
```
Comment (ArneGudermann) [inline deploy/emails/user_password_recovery.email:31]:
```suggestion
```
This is obsolete the bone is hidden.

**Review by ArneGudermann — APPROVED** [review]

**Pull request merged**